app/views/layouts/application.html.haml

```diff
@@ -4,8 +4,8 @@
 %meta{:content => "width=device-width,initial-scale=1", :name => "viewport"}/
 = csrf_meta_tags
 = csp_meta_tag
-= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
-= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload'
+= stylesheet_pack_tag 'styles/application', media: 'all', 'data-turbolinks-track': 'reload'
+%link{:href => "https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap", :rel => "stylesheet"}/
```

Comment:
I think it's perfectly legit to include external fonts in the layout head this way. However, you might also want to look at `@font-face`, which is here: https://gist.github.com/anotheruiguy/7379570